plugins/fclrave/fclspace.h

```diff
@@ -152,6 +152,7 @@ class FCLSpace : public boost::enable_shared_from_this<FCLSpace>
 
         KinBodyWeakPtr _pbody;
         int nLastStamp;  ///< KinBody::GetUpdateStamp() when last synchronized ("is transform up to date")
+        int nLastLinkReloadStamp; ///< KinBody::GetUpdateStamp() when we last ran ReloadKinBodyLinks
```

Comment:
I would just initialize all these `int` members in case of missed initialization